archaius-core/src/main/java/com/netflix/config/sources/URLConfigurationSource.java

```diff
@@ -180,8 +181,17 @@ public PollResult poll(boolean initial, Object checkPoint)
         for (URL url: configUrls) {
             Properties props = new Properties();
             InputStream fin = url.openStream();
-            props.load(fin);
-            fin.close();
+            InputStreamReader reader = new InputStreamReader(fin, "UTF-8");
```

Comment:
This is a biggie. I didn't notice this before because we didn't run into any CJK values before. In current code path, the most frequently used file properties loading is through PropertiesConfiguration in Apache Commons Configurations. But I like not depending on that and doing it this way. The only comment I have here is as above. Consolidate the code into a function in ConfigurationUtils so we can re-use.